DEV.md

```diff
@@ -45,36 +45,93 @@ $ make watch
 
 ## Testing
 
-To execute the test suite runtime dependencies must be running.
-```
-$ make start-deps
-docker-compose up -d
-(...)
-```
-
-Everytime the dependencies are re-started the environment must be reloaded via `direnv reload`. When using the [make command](./Makefile) this will happen automatically.
-
 To run the test suite you can run:
-```
+
+```bash
 $ make test
-(...)
-$ echo $?
-0
 ```
-or
-```
-# to run unit tests
+To execute the test suite [runtime dependencies](#runtime-dependencies) must be running.
+
+### Run unit tests
+
+```bash
+$ yarn test:unit
+# or
 $ make unit
+```
+
+Runtime dependencies are not required
+
+### Run integration tests
 
-# to run integration test
+To execute the integration tests [runtime dependencies](#runtime-dependencies) must be running.
+
+```bash
+$ yarn test:integration
+# or
 $ make integration
 ```
 
-The tests are *not* idempotent (yet) so currently to re-run the tests, run:
+The  integration tests are *not* fully idempotent (yet) so currently to re-run the tests, run:
 ```
 $ make reset-integration
 ```
 
+### Run specific test file
+
+To execute a specific test file:
+
+#### Unit
+
+Example to run `test/unit/config.spec.ts`
+
+```bash
+$ TEST=config yarn test:unit
```

Comment:
I think you've actually deleted that particular file right?

Reply:
:man_facepalming:

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/340/commits/566b8414e428184b880baea8d47b622d71a857d7